servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -209,6 +210,30 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
     }
   }
 
+  private static void applyPitestPlugin(Project project) {
+    project.configure(project) {
+      pluginManager.apply("info.solidsoft.pitest")
+
+      pitest {
+        pitestVersion = project.properties["pitestVersion"]
+        junit5PluginVersion = project.properties["pitestPluginJunit5Version"]
+
+        timestampedReports = false
+
+        if (project.ext.isCiBuild) {
+          outputFormats = ['XML']
+        } else {
+          outputFormats = ['HTML']
+        }
+        failWhenNoMutations = false
+      }
+
+      tasks.withType(PitestTask).all {
+        group = "verification"
```

Comment:
Looks like it's already in `verification` group, we can remove this

Reply:
I will comment it out for now.

Reply:
On further investigation this snippet was only adding the task to the correct group for the `gradlew tasks` command and not configuring the task for execution. The task still needs to be invoked manually. For automatic execution it would be necessary to add it as a dependency or add it as a dependency to the quality task.